daphne_worker/src/durable/leader_col_job_queue.rs

```diff
@@ -24,10 +24,10 @@ pub(crate) const DURABLE_LEADER_COL_JOB_QUEUE_FINISH: &str =
 pub(crate) const DURABLE_LEADER_COL_JOB_QUEUE_GET_RESULT: &str =
     "/internal/do/leader_col_job_queue/get_result";
 
-#[derive(Debug, Deserialize, Serialize)]
-pub(crate) struct CollectionJobResult {
-    pub(crate) collect_id: Id,
-    pub(crate) collect_resp: CollectResp,
+#[derive(Deserialize, Serialize)]
+pub(crate) enum CollectionJobQueueResult {
+    Ok(Id),
+    ErrBatchOverlap,
```

Comment:
This is no longer returned. Do we still need CollectionJobQueueResult? It seem slike this can be revereted

Reply:
Great point, I've removed this.